app/src/main/java/com/zeapo/pwdstore/autofill/AutofillService.kt

```diff
@@ -492,50 +490,48 @@ class AutofillService : AccessibilityService() {
             data = resultData!!
             resultData = null
         }
-        var `is`: InputStream? = null
-        try {
-            `is` = FileUtils.openInputStream(items[lastWhichItem])
-        } catch (e: IOException) {
-            e.printStackTrace()
+
+        var inputStream: InputStream? = null
+        withContext(Dispatchers.IO) {
+            inputStream = FileUtils.openInputStream(items[lastWhichItem])
```

Comment:
Bring back the try catch, otherwise the service will crash when an exception occurs here.